src/views/SystemIntake/Documents/UploadForm.tsx

```diff
@@ -268,6 +278,60 @@ const UploadForm = ({ type = 'requester' }: UploadFormProps) => {
             </Fieldset>
           </FormGroup>
 
+          {/* display for admins only when accessed from admin view */}
+          {type === 'admin' && user.isITGovAdmin(groups, flags) && (
+            <Controller
+              name="sendNotification"
+              control={control}
+              render={({ field, fieldState: { error } }) => (
+                <FormGroup className="margin-top-3" error={!!error}>
+                  <Label htmlFor={field.name}>
```

Comment:
Since there are multiple radio inputs within this field, it should probably be wrapped in a `Fieldset` component with a `legend` instead of `label`.

The "version" field above this is an example of using the `Fieldset` component with radio inputs.